helm-chart/ray-cluster/values.yaml

```diff
@@ -102,12 +103,13 @@ worker:
   volumeMounts:
     - mountPath: /tmp/ray
       name: log-volume
+  sidecarContainers: {}
```

Comment:
Hmm, maybe my Helm is rusty, but aren't sidecarContainers a list?

Would it be correct to have the following:
```
sidecarContainers: []
```
The [prometheus chart](https://github.com/helm/charts/blob/master/stable/prometheus/values.yaml#L565) just has 
```
sidecarContainers:
```

Reply:
Updated. The value of `{{- if .Values.head.sidecarContainers }}` will be false for both `[]` and `{}`, but `{}` may cause users to feel confused. Hence, it is a good idea to update it to `[]`.

Reply:
Perfect, thanks!